docs/fundamentals/jcloud/advanced.md

```diff
@@ -78,3 +78,15 @@ executors:
 ```{figure} external-executors-multiple.png
 :width: 70%
 ```
+## Deploy with specific `jina` version
+
+When deploying Flow to `jcloud`, the default `jina` version would be used. If for any reason you'd like your Flow to be built with a specific `jina` version, you can do so by using `version` arg while deploying the Flow.
```

Comment:
I'd avoid talking about the default jina version on jcloud, as it depends on when we can merge the dependabot PR.

```suggestion
To manage `jina` version while deploying a Flow to `jcloud`, you can pass `version` arg in the Flow yaml.
```

Reply:
makes sense